userspace/libsinsp/sinsp.cpp

```diff
@@ -55,7 +55,7 @@ limitations under the License.
  * big enough to prevent the queue to ever fill-up in standard circumstances,
  * while at the same time avoiding it growing uncontrollably in case of anomalies.
 */
-#define DEFAULT_ASYNC_EVENT_QUEUE_SIZE 1000
+#define DEFAULT_ASYNC_EVENT_QUEUE_SIZE 30000
```

Comment:
This needs to be reverted to the old value right? As per @Molter73 comment :)

Reply:
Given that the current size is becoming restrictive I think we should increase the default size and use the runtime setter if the current size has a performance impact.
However I don't know what's the most common use case for adopters, so I'll let the maintainers take a decision on this.
cc @jasondellaluce

Reply:
I'm ok in keeping the old default value if that's something that matters for adopters here! As long as we have the setter we should be good